src/components/Typography/Typography.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+import { StyleSheet, Text } from "react-native";
```

Comment:
you can import this text using

Text as RnText 


That way whenever we see text, we know that we have to replace with one of ours